### PR TITLE
Optional Group Membership Notifications #4585

### PR DESCRIPTION
**Pull request by MasterHans**
**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [x] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

**Other information:**
1) Added notify_users column to group table.
2) Add checkbox to group settings

### CODE REVIEW

**Review by luke- — CHANGES_REQUESTED** [review]
Migration file missing?

Please also always add changes to the CHANGELOG file.

File: protected/humhub/modules/admin/views/group/edit.php [hunk 1]
```diff
@@ -38,6 +38,21 @@
...
 
+    <strong><?= Yii::t('AdminModule.user', 'Users notification'); ?></strong>
+    <br>
+    <br>
+
+    <?= $form->field($group, 'notify_users')->checkbox(); ?>
+    <div class="help-block">
+    <?= Yii::t(
+        'AdminModule.user',
+        'Notify users about adding to the group'
```
Comment (buddh4) [inline protected/humhub/modules/admin/views/group/edit.php:49]: Notify users when adding them to this group

**Review by buddh4 — CHANGES_REQUESTED** [review]

File: protected/humhub/modules/admin/views/group/edit.php (hunk 1 above)
Comment (MasterHans) [reply protected/humhub/modules/admin/views/group/edit.php]: Done!

Comment (luke-) [reply protected/humhub/modules/admin/views/group/edit.php]: @MasterHans I've moved this to the attribute hints: https://github.com/humhub/humhub/pull/4643/files#diff-a48b33b5fa0409f1ae133b00086682c396c4f4ea470d30d47823c15231284e9dR99

**Review by luke- — APPROVED** [review]

**Comment by luke-** [comment]
@MasterHans Please always add the PR to the CHANGELOG file.

**Comment by MasterHans** [comment]
> @MasterHans Please always add the PR to the CHANGELOG file.

OK, thank you

**Pull request merged**